chart/values.schema.json

```diff
@@ -857,6 +857,15 @@
                         "description": "Type **as string** of secret E.G. Opaque, kubernetes.io/dockerconfigjson, etc.",
                         "type": "string"
                     },
+                    "labels": {
+                        "description": "Labels for the secret",
+                        "type": "object",
+                        "additionalProperties": {
+                            "description": "Label key",
```

Comment:
I'm not sure we need this description?

Reply:
@jedcunningham probably not, but I am getting the following error in static checks when I try to remove `description` or `default` from here:

```
Processing file:  chart/values.schema.json
'description' is a required property

Failed validating 'required' in schema['properties']['properties']['additionalProperties']['allOf'][0]['additionalProperties']['additionalProperties']['additionalProperties']['additionalProperties']['then']:
    {'required': ['description', 'default']}

On instance['properties']['extraConfigMaps']['additionalProperties']['properties']['labels']['additionalProperties']:
    {'type': 'string'}
'default' is a required property

Failed validating 'required' in schema['properties']['properties']['additionalProperties']['allOf'][0]['additionalProperties']['additionalProperties']['additionalProperties']['additionalProperties']['then']:
    {'required': ['description', 'default']}

On instance['properties']['extraConfigMaps']['additionalProperties']['properties']['labels']['additionalProperties']:
    {'type': 'string'}
```

Looks like one possible way to fix it is to remove the lines 7-11 from https://github.com/apache/airflow/blob/main/chart/values_schema.schema.json :

```
            "additionalProperties": {
                "additionalProperties": {
                    "$ref": "#/definitions/leafs"
                }
            },
```

Should I just remove these lines? Or should I leave it as it is? Or maybe there are better ways to fix it?

Reply:
Removed this `description` - after some thinking looks like there is a way to remove `description`  and `default` from here without breaking or making useless other checks from [values_schema.schema.json](https://github.com/apache/airflow/blob/main/chart/values_schema.schema.json) .